src/libraries/apicompat/ApiCompatBaseline.NetCoreAppLatestStable.xml

```diff
@@ -1,22 +1,4 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <!-- https://learn.microsoft.com/en-us/dotnet/fundamentals/package-validation/diagnostic-ids -->
 <Suppressions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
-  <Suppression>
```

Comment:
Why did you decide to remove these?

Reply:
These suppressions were flagged as unnecessary now as the `RequiresPreviewFeaturesAttribute` attribute was removed from `RefreshMemoryLimit` between RC1 and GA.

Reply:
Ah that's neat. You are also told when a suppression is unnecessary. Cool.

Reply:
Yes, that's a new feature in APICompat, part of .NET 8.